service/src/main/java/uk/gov/hmcts/reform/fpl/enums/CMOActionStatus.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum CMOActionStatus {
+    SEND_TO_ALL_PARTIES("Yes, it can be sealed and sent to parties", "case-management-order.pdf"),
+    LOCAL_AUTHORITY_CHANGES("No, local authority needs to make changes", "draft-case-management-order.pdf"),
+    SELF_REVIEW("No, I need to make changes", "draft-case-management-order.pdf");
+
+    private final String actionedStatusValue;
+    private final String documentTitle;
```

Comment:
IMO document title is not directly associated with values of that enum and for such reason other component should decide that is the document title for given typem of action.

My intention is to keep classes single responsibility, focused on one thing only and having one reason for change.